client/layout/sidebar/custom-icon.jsx

```diff
@@ -13,20 +13,21 @@
  */
 import React from 'react';
 
-const SidebarCustomIcon = ( { alt, className, icon, ...rest } ) => {
+const SidebarCustomIcon = ( { icon, ...rest } ) => {
 	if ( ! icon ) {
 		return null;
 	}
 
 	if ( icon.indexOf( 'data:image' ) === 0 || icon.indexOf( 'http' ) === 0 ) {
+		const imgStyle = `url("${ icon }")`;
+		const imgStyles = { backgroundImage: imgStyle, maskImage: imgStyle, WebkitMaskImage: imgStyle };
+
 		return (
-			<img
-				alt={ alt || '' }
```

Comment:
From [HTMLImageElement.alt](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt): 

> Perhaps the most important reason to use the alt tag is to support accessibility, as the alt text may be used by screen readers and other assistive technologies to help differently-abled users make full use of your content. It will be read aloud or sent to the touch device, for example, to support visually impaired users.

Since the `alt` attribute is going to be removed, and the element replaced with a `span`, should we add `aria-label` with the alt value or something like that?

Reply:
Yes, good point. I don't think I've seen an `alt` attribute used yet, but that doesn't mean there won't be